emmo-mechanical-testing.owl

```diff
@@ -6,6 +6,7 @@
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xml="http://www.w3.org/XML/1998/namespace"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema#"
+     xmlns:emmo="http://emmo.info/emmo#"
```

Comment:
I would do this myself but I think write permission on your pull requests is not enabled.

Can you remove this line? In the end it is not being used.